test/bases/renku_data_services/data_api/test_resource_pools.py

```diff
@@ -24,6 +24,8 @@
     "quota": {"cpu": 100, "memory": 100, "gpu": 0},
     "default": False,
     "public": True,
+    "idle_threshold": 86400,
+    "hibernation_threshold": 99999,
```

Comment:
Shouldn't we check somewhere (maybe in `test_get_single_pool_quota` or a new test) to see if these values are correctly set?

Reply:
I added an extra test